client/src/components/Cars/car-card.component.js

```diff
@@ -0,0 +1,105 @@
+import React, { useContext } from "react";
+import Grid from "@material-ui/core/Grid";
+import Paper from "@material-ui/core/Paper";
+import { makeStyles, useTheme } from "@material-ui/core/styles";
+import { Typography, Button } from "@material-ui/core";
+import { addToShoppingCart } from "../../utils/shoppingCart";
+import { ShoppingCartContext } from "../ShoppingCartContext";
+
+const useStyles = makeStyles((theme) => ({
+  root: { border: "1px" },
+  paper: { padding: "15px" },
+  carDetail: {
+    textAlign: "left",
+    display: "flex",
+    margin: "10px 0",
+  },
+  carIcon: {
+    fontSize: "100px",
+  },
+  carImage: {
+    height: "100%",
+    maxHeight: "200px",
+    maxWidth: "100%",
+  },
+
+  button: {
+    color: "white",
+  },
+}));
+
+const CarCard = (props) => {
+  const theme = useTheme();
+  const classes = useStyles(theme);
+
+  const [shoppingCart, setShoppingCart] = useContext(ShoppingCartContext);
+
+  const handleCarSelection = (e) => {
+    e.preventDefault();
```

Comment:
I actually don't think you need this. Typically this is needed for things like `type="submit"` buttons to override the default behaviour